webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/components/fields/DownloadUtility.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  FormGroup,
+  FormSelect,
+  FormSelectOption,
+} from '@patternfly/react-core';
+
+import { translate as __ } from '../../../../../common/I18n';
+
+export const DownloadUtilities = ['curl', 'wget'];
+const DownloadUtility = ({
+  downloadUtility,
+  handleDownloadUtility,
+  isLoading,
+}) => (
+  <FormGroup label={__('Download utility')} fieldId="reg_download_utility">
+    <FormSelect
+      ouiaId="reg_download_utility"
+      value={downloadUtility}
+      onChange={v => handleDownloadUtility(v)}
+      className="without_select2"
+      id="reg_download_utility"
+      isDisabled={isLoading}
+    >
+      {DownloadUtilities.map(item => (
+        <FormSelectOption key={item} value={item} label={item} />
+      ))}
+    </FormSelect>
+  </FormGroup>
+);
+
+DownloadUtility.propTypes = {
+  downloadUtility: PropTypes.oneOf([
+    DownloadUtilities.curl,
+    DownloadUtilities.wget,
```

Comment:
Does this actually work? Shouldn't this be `PropTypes.oneOf(DownloadUtilities)`?